src/mscorlib/facade/mscorlib.csproj

```diff
@@ -97,6 +88,42 @@
     <SeedTypePreference Include="System.Console">
       <Assembly>System.Console</Assembly>
     </SeedTypePreference>
+    <SeedTypePreference Include="System.IO.BinaryReader">
```

Comment:
There were old System.IO facades being picked up by some CoreCLR tests. I tried updating project.jsons for CoreCLR tests to use latest corefx to make things match, but it broke other things.

So I have just done this conservative fix for now to get this change in and unblock the CI.